frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+    cy.login();
+    cy.install();
+    cy.visit('/dashboards/object-service');
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Check that noobaa dashboard is opening and links available.', () => {
+    cy.byLegacyTestID('system-name-mcg')
+      .invoke("attr", "href")
+      .then(href => {
+        cy.request(href)
+          .then((response) => {
+            expect(response.status).to.eq(200)
+            expect(response.body).to.contain(
+              'NooBaa Management Console')
+          });
```

Comment:
```suggestion
            expect(response.status).toEqual(200)
            expect(response.body).contains(
              'NooBaa Management Console')
          });
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/7570/pull-ci-openshift-console-master-frontend/1340992172702306304#1:build-log.txt%3A1485